tools/get_kubectl_kind.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euo pipefail
+source build.env
+
+# This script downloads kind binary that are
+# used as part of the vtop test environment,
+# and places them in vitess/bin/.
```

Comment:
Is that file supposed to be used in the CI? If so, I recommend we use a GitHub Action to get Kind directly to ease maintenance. For instance:
- https://github.com/engineerd/setup-kind
- https://github.com/marketplace/actions/kind-cluster

Reply:
This is a better alternate, but we'll add the workflow back when we are in buildkite. I'll look for alternates to the file then